examples/cq_tutorial.py

```diff
@@ -7,11 +7,11 @@
 diameter = 22.0
 padding = 12.0
 
-model = cq.Workplane("XY").box(height, width, thickness) \
-    .faces(">Z").workplane().hole(diameter) \
-    .faces(">Z").workplane() \
-    .rect(height - padding, width - padding, forConstruction=True) \
-    .vertices().cboreHole(2.4, 4.4, 2.1) \
+model = (cq.Workplane("XY").box(height, width, thickness)
```

Comment:
Thanks!

Could you add a line break here too, before `.box`? :)

Reply:
Done!